src/modules/article/dto/received/get-by-gender.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString } from 'class-validator';
+import { GenderEnum } from '../../../../shared/enums/gender.enum';
+
+export class GetByGenderDto {
+  @ApiProperty({ description: 'user gender', example: GenderEnum.MALE })
+  @IsString()
+  @IsNotEmpty()
+  gender: GenderEnum;
```

Comment:
you need to validate enum, 
and gender: string

Reply:
done